src/components/InputsBox.jsx

```diff
@@ -1,7 +1,17 @@
-import { Button, Grid, TextField } from '@mui/material';
+import { Button, Grid, TextField, Box } from '@mui/material';
 import TextareaAutosize from '@mui/material/TextareaAutosize';
+import { useEffect, useState } from 'react';
 
 export default function InputsBox() {
+  const [selectedImage, setSelectedImage] = useState(null);
```

Comment:
No need for `const [selectedImage, setSelectedImage] = useState(null);` and connect it to useEffect.
Create a function called `onImageChange` for example, pass this function the image file coming from input `onChange`. this function should check if the file is actually an image and no other type of file. if everything is correct than call `setImageUrl(URL.createObjectURL(selectedImage));`.